adapterearthengine/services/dataset_service.py

```diff
@@ -4,19 +4,15 @@
 
 
 class DatasetService(object):
-
     @staticmethod
-    def execute(config):
-        response = request_to_microservice(config)
-        if not response or response.get('errors'):
-            raise DatasetNotFound(message='Dataset not found')
-        dataset = response.get('data', None).get('attributes', None)
+    def execute(uri, api_key):
+        response = request_to_microservice(uri=uri, api_key=api_key, method="GET")
+        if not response or response.get("errors"):
+            raise DatasetNotFound(message="Dataset not found")
+        dataset = response.get("data", None).get("attributes", None)
```

Comment:
I know it's not the focus of your PR, but I just want to point out that this doesn't really work the way it's supposed to. It's asking for an AttributeError, because None doesn't have a .get method. Not a blocker because you didn't touch it (I assume this was a linter). But something to consider fixing in the future.

Reply:
Thanks for flagging so we address this in future cleanup.